parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -583,13 +562,12 @@ configAddons() {
 {{- if HasNSeriesSKU}}
 {{- /* installNvidiaDrivers is idempotent, it will uninstall itself if it is already installed, and then install anew */}}
 installNvidiaDrivers() {
-  NVIDIA_DKMS_DIR="/var/lib/dkms/nvidia/${GPU_DV}"
-  KERNEL_NAME=$(uname -r)
-  if [ -d $NVIDIA_DKMS_DIR ]; then
-    dkms remove -m nvidia -v $GPU_DV -k $KERNEL_NAME
+  local d="/var/lib/dkms/nvidia/${GPU_DV}" k log_file="/var/log/nvidia-installer-$(date +%s).log"
+  k=$(uname -r)
+  if [ -d $d ]; then
```

Comment:
Shortening these variable names makes things less readable, but these changes are mostly localized so I suppose it's tolerable. Whatever is necessary to save bytes.